ibm_db_ctx.py

```diff
@@ -0,0 +1,25 @@
+import ibm_db
+
+class Db2connect:
+    """Context manager to handle connections to DB2."""
+    #__cxn: Optional['IBM_DBConnection'] 
+    
+    def __init__(self, dsn: str, username: str, password: str) -> None:
+        """Instantiate a DB2 connection."""
+        #print("init method called")
+        self.__dsn = dsn
+        self.__username = username
+        self.__password = password
+        self.__cxn = None
+
+    def __enter__(self) -> 'IBM_DBConnection':
+        """Connect to DB2."""
+        self.__cxn = ibm_db.connect(self.__dsn, '', '')
```

Comment:
Please put username and password here

Reply:
> Please put username and password here

Username and password will be there in DSN only. 
For cataloged connection, we need to give database, username and password [ For this new test suite added for both cataloged and uncatalogued connection].

Reply:
Ok. Then why are we taking username and password in the constructor then. I don't see those values being use anywhere?

Reply:
The use of username and password depends on what type of connection we are trying to. 
When we go for an uncatalogued connection, we will pass an empty string only for username and password field.
Please refer to:  https://github.com/ibmdb/python-ibmdb/wiki/APIs#ibm_dbconnect

Reply:
ok